packages/cli/src/lib/redis/api.ts

```diff
@@ -84,22 +107,19 @@ export default (app: string, database: string | undefined, json: boolean, heroku
       return onResponse
     },
 
-    async getRedisAddon(addonsList?: Heroku.AddOn[]) {
-      if (addonsList === undefined) {
-        const {body: list} = await heroku.get<Heroku.AddOn[]>(`/apps/${app}/addons`)
-        addonsList = list
-      }
+    async getRedisAddon() {
```

Comment:
I noticed you removed the `addonsList` parameter here.. eventually I think we will need it because it's used here: https://github.com/heroku/cli/blob/main/packages/redis-v5/commands/promote.js#L16

Reply:
Yes, I noticed it, but if you take a closer look, it's called with the same default URL that `getRedisAddon` uses. I don't know why, but it's completely superfluous.

Reply:
oh good catch!